plugins/firebase.js

```diff
@@ -59,6 +59,75 @@ const fireDb = {
     }
     return introTexts
   },
+  getEvents: async () => {
+    const websites = await fireDb.getWebsites()
+    const events = {}
+    for (const website of websites) {
+      const websiteData = await db
+        .collection(webCollection)
+        .doc(website)
+        .collection('Events')
+        .get()
+      events[website] = await websiteData.docs.map(doc => {
+        const data = doc.data()
+        return {
+          id: doc.id,
```

Comment:
don't you need || -1 here as well?

Reply:
This is in case the order is passed a string that can't be parsed to an int. We could have it so that the event can't be saved unless the order is parsable.

Reply:
OK sounds nice